src/css/application.css

```diff
@@ -303,6 +305,18 @@ body {
   flex: 1 0 0;
 }
 
+.top-bar__snapshot {
```

Comment:
Doesn’t look like this is used?

Reply:
This takes the margin off the right of the snapshot button

Reply:
in `SnapshotButton.jsx`